src/main/java/com/hubspot/jinjava/lib/filter/DateTimeFormatFilter.java

```diff
@@ -11,8 +11,9 @@
     value = "Formats a date object",
     params = {
         @JinjavaParam(value = "value", defaultValue = "current time", desc = "The date variable or UNIX timestamp to format"),
-        @JinjavaParam(value = "format", defaultValue = StrftimeFormatter.DEFAULT_DATE_FORMAT, desc = "The format of the date determined by the directives added to this parameter"),
-        @JinjavaParam(value = "timezone", defaultValue = "utc", desc = "Time zone of output date")
+        @JinjavaParam(value = "format", type = "string", defaultValue = StrftimeFormatter.DEFAULT_DATE_FORMAT, desc = "The format of the date determined by the directives added to this parameter"),
+        @JinjavaParam(value = "timezone", type = "string", defaultValue = "UTC", desc = "Time zone of output date"),
+        @JinjavaParam(value = "languageTag", type = "string", defaultValue = "us", desc = "The language to use when formatting the datetime")
```

Comment:
Can the parameter just be named "language"?

Reply:
What's a list of valid languages?

Reply:
shouldn't it be locale? UK english != US English kinda thing?